src/ray/core_worker/transport/direct_actor_transport.cc

```diff
@@ -406,6 +452,9 @@ void CoreWorkerDirectTaskReceiver::HandleTask(
   auto reject_callback = [send_reply_callback]() {
     send_reply_callback(Status::Invalid("client cancelled stale rpc"), nullptr, nullptr);
   };
+  auto skip_callback = [send_reply_callback]() {
+    send_reply_callback(Status::OK(), nullptr, nullptr);
```

Comment:
Could we just put this logic into the normal accept callback instead of passing a separate skip callback?